### ISSUE TITLE
make not working

### ISSUE THREAD

**Issue by JohnGriffiths** [mindcruzer/pelican-encrypt-content#3]
Hi. 

Thanks for the plugin. This is exactly what I need. 

However it's not working for me yet. 

I think I have followed the steps correcting, but am getting the following error when I type `make html`:

>>CRITICAL: encrypt-content-content.html
>>make: *** [html] Error 1

Any idea what the problem might be?

Ta, 

john

**Comment by mindcruzer** [comment]
Likely something related to recent changes in Pelican (I made this when v3.3 was current). I will check it out and get back to you.

**Comment by mindcruzer** [comment]
Did you copy the `encrypt-content-content.html` file to your theme's `templates` directory? This error is occurring because pelican can't find the file.

**Comment by JohnGriffiths** [comment]
Ok you are correct - copying that file ( + 'encrypt-content-scripts.html' )
seems to have done the job.

Still not quite working though.

The make html and devserver appears to finish without problems, but when I
click on a link to an encrypyted page, and enter in the password in the box
that comes up, it just hangs.

Any idea why this might be?


Thanks,

john




On 26 February 2015 at 03:56, Sean Stewart <notifications@github.com> wrote:

> Did you copy the encrypt-content-content.html file to your theme's
> templates directory? This error is occurring because pelican can't find
> the file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mindcruzer/pelican-encrypt-content/issues/3#issuecomment-76000864>
> .
>



-- 

Dr. John Griffiths

Post-Doctoral Research Fellow

Rotman Research Institute, Baycrest

Toronto, Canada

and

Visiting Research Fellow

School of Physics

University of Sydney

**Comment by mindcruzer** [comment]
It sounds like the script file isn't being loaded. Did you merge `themes/static` as well?

**Comment by JohnGriffiths** [comment]
Solved! Thank you.

I copied the files over from the static directory manually, and now it's
working.

Previously I did this using an rsync merge; that must have gone wrong
somehow the first time.

This is going to be very useful :)


Ta,


john








On 26 February 2015 at 10:14, Sean Stewart <notifications@github.com> wrote:

> It sounds like the script file isn't being loaded. Did you merge
> themes/static as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mindcruzer/pelican-encrypt-content/issues/3#issuecomment-76082450>
> .
>



-- 

Dr. John Griffiths

Post-Doctoral Research Fellow

Rotman Research Institute, Baycrest

Toronto, Canada

and

Visiting Research Fellow

School of Physics

University of Sydney
